src/BitPaySDK/Exceptions/PayoutNotificationException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace BitPaySDK\Exceptions;
+
+
+use Exception;
+
+class PayoutNotificationException extends PayoutException
+{
+    private $bitPayMessage = "Failed to send payout notification";
+    private $bitPayCode    = "BITPAY-PAYOUT-NOTIFICATION";
+    protected $apiCode;
+
+    /**
+     * Construct the PayoutNotificationException.
+     *
+     * @param string $message [optional] The Exception message to throw.
+     * @param int    $code    [optional] The Exception code to throw.
+     * @param string $apiCode [optional] The API Exception code to throw.
+     */
+    public function __construct($message = "", $code = 1212, Exception $previous=NULL, $apiCode = "000000")
```

Comment:
sdk codes are 3 digits only, make sure you have created new ones for this resource and that there are no duplicates